src/App.tsx

```diff
@@ -1,38 +1,37 @@
-import React from 'react';
+import { FC, useState } from 'react';
 import './App.scss';
+import { Tabs } from './components/Tabs/Tabs';
+import Tab from './types/Tab';
 
 export const tabs = [
   { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
-export const App: React.FC = () => {
+export const App: FC = () => {
+  const [choosedTab, setChoosedTab] = useState(tabs[0]);
+
+  const onTabSelected = (tab: Tab) => {
+    if (tab.id !== choosedTab.id) {
+      setChoosedTab(tab);
+    }
+  };
+
   return (
     <div className="section">
       <h1 className="title">
-        Selected tab is&nbsp;Tab 1
+        Selected tab is&nbsp;
+        {choosedTab.title}
```

Comment:
here, you'd rather use interpolation – it is easier to implement + it will make it look simpler